.travis.yml

```diff
@@ -10,17 +10,9 @@ matrix:
     # Nightly builds run the tests/examples end-to-end test suite.
     # Does not include Windows since Projects/Models are mostly not compatible with Windows as of now.
     - stage: Nightly
-    - stage: Small
```

Comment:
corresponding section for `TRAVIS_BUILD_STAGE_NAME` from `install:` line 138-141 (old version) can be cleaned out. Also remove `"$TRAVIS_BUILD_STAGE_NAME" == "Small"` in `./travis/stage-python3.sh`

Reply:
Removed!